hr_timesheet_report_not_valued/i18n/es.po

```diff
@@ -0,0 +1,23 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* hr_timesheet_report_not_valued
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-06-14 09:46+0000\n"
+"PO-Revision-Date: 2021-06-14 11:47+0200\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 2.4.3\n"
+"Last-Translator: \n"
+"Language: es_ES\n"
+
+#. module: hr_timesheet_report_not_valued
+#: model:ir.actions.report,name:hr_timesheet_report_not_valued.timesheet_not_valued_report
+msgid "Timesheet Entries Not Valued"
+msgstr "Detalle de los partes de horas no valorado"
```

Comment:
v15 translations have changed at this point, and we have this inconsistency:

![imagen](https://github.com/solvosci/slv-timesheet/assets/12749832/73a7ab71-e624-42d4-bc5c-e456d9f0355d)

So, a better translation, for coherence, should be the following:
```suggestion
msgstr "Entradas de los partes de horas no valoradas"
```